src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/details.html.twig

```diff
@@ -23,46 +23,90 @@
  *#}
 
+{% set statusTitle %}
+  {{ 'Status'|trans({}, 'Admin.Global') }} ({{ orderForViewing.history.statuses|length }})
+{% endset %}
+
+{% set documentsTitle %}
+  {{ 'Documents'|trans({}, 'Admin.Orderscustomers.Feature') }} ({{ orderForViewing.documents.documents|length }})
+{% endset %}
+
+{% set carriersTitle %}
+  {{ 'Carriers'|trans({}, 'Admin.Shipping.Feature') }} ({{ orderForViewing.shipping.carriers|length }})
+{% endset %}
+
+{% set merchantReturnsTitle %}
+  {{ 'Merchandise returns'|trans({}, 'Admin.Orderscustomers.Feature') }} ({{ orderForViewing.returns.orderReturns|length }})
+{% endset %}
+
 <div class="mt-2">
-  <ul class="nav nav nav-tabs" role="tablist">
+  <ul class="nav nav nav-tabs d-print-none" role="tablist">
     <li class="nav-item">
       <a class="nav-link active show" id="historyTab" data-toggle="tab" href="#historyTabContent" role="tab" aria-controls="historyTabContent" aria-expanded="true" aria-selected="false">
         <i class="material-icons">history</i>
-        {{ 'Status'|trans({}, 'Admin.Global') }} ({{ orderForViewing.history.statuses|length }})
+        {{ statusTitle }}
       </a>
     </li>
     <li class="nav-item">
       <a class="nav-link" id="orderDocumentsTab" data-toggle="tab" href="#orderDocumentsTabContent" role="tab" aria-controls="orderDocumentsTabContent" aria-expanded="true" aria-selected="false">
         <i class="material-icons">note</i>
-        {{ 'Documents'|trans({}, 'Admin.Orderscustomers.Feature') }} ({{ orderForViewing.documents.documents|length }})
+        {{ documentsTitle }}
       </a>
     </li>
     <li class="nav-item">
       <a class="nav-link" id="orderShippingTab" data-toggle="tab" href="#orderShippingTabContent" role="tab" aria-controls="orderShippingTabContent" aria-expanded="true" aria-selected="false">
         <i class="material-icons">local_shipping</i>
-        {{ 'Shipping'|trans({}, 'Admin.Catalog.Feature') }} ({{ orderForViewing.shipping.carriers|length }})
+        {{ carriersTitle }}
       </a>
     </li>
     <li class="nav-item">
       <a class="nav-link" id="orderReturnsTab" data-toggle="tab" href="#orderReturnsTabContent" role="tab" aria-controls="orderReturnsTabContent" aria-expanded="true" aria-selected="false">
         <i class="material-icons">replay</i>
-        {{ 'Merchandise returns'|trans({}, 'Admin.Orderscustomers.Feature') }} ({{ orderForViewing.returns.orderReturns|length }})
+        {{ merchantReturnsTitle }}
       </a>
     </li>
   </ul>
 
   <div class="tab-content">
-    <div class="tab-pane fade show active" id="historyTabContent" role="tabpanel" aria-labelledby="historyTab">
-      {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/history.html.twig' %}
+    <div class="tab-pane d-print-block fade show active" id="historyTabContent" role="tabpanel" aria-labelledby="historyTab">
+      {% embed '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/details_card.html.twig' %}
```

Comment:
This changes the display of the tabs right? not just for print view but in general (since it includes a card)
Is it what is expected in the design?

Reply:
Unless the card is only visible for print, and in screen view it seems as if there is no change thanks to `card--details` ?

Reply:
It just force the card to use `display: block` in print view so it take the full width space

Reply:
So is the card really necessary to do that? If the aim is only to have a `display: block;` on tabs it could be managed by adding it on `.tab-pane` directly If it was as simple as that then we could also apply it for hooks tabs
But I have the feeling the card is used so that the tab title may be printed close to the tab content, just a guess though Although I get the idea what bothers me is that this solution is a bit complex and won't work for hooks tabs.. (and I hope it doesn't modify the screen display)